shimming-toolbox/shimmingtoolbox/cli/b0shim.py

```diff
@@ -736,10 +750,10 @@ def realtime_dynamic(fname_fmap, fname_anat, fname_mask_anat_static, fname_mask_
     # Load the coils
     list_coils_static = _load_coils(coils_static, scanner_coil_order_static, fname_sph_constr, nii_fmap,
                                     options['scanner_shim'], json_fm_data['Manufacturer'],
-                                    json_fm_data['ManufacturersModelName'])
+                                    json_fm_data['ManufacturersModelName'], isocenter_fm)
     list_coils_riro = _load_coils(coils_riro, scanner_coil_order_riro, fname_sph_constr, nii_fmap,
                                   options['scanner_shim'], json_fm_data['Manufacturer'],
-                                  json_fm_data['ManufacturersModelName'])
+                                  json_fm_data['ManufacturersModelName'], isocenter_fm)
```

Comment:
At this point, wouldn't it be better to give the json_fm_data as input and get the specifics in the Coil() object? Lines 739 to 744 could also go inside the coil object unless you need the isocenter elsewhere.

Reply:
Fixed in [9cec368](https://github.com/shimming-toolbox/shimming-toolbox/pull/539/commits/9cec3681745602df2ee4c9edfba0108c9ede531d)